### PR TITLE
Simple Spring Queries

### PR DESCRIPTION
**Pull request by yschimke**
example command line

```
rsocket-cli --stream --route 'searchTweets' -i "{}" ws://localhost:8080/rsocket
```

### CODE REVIEW

File: src/main/kotlin/io/rsocket/cli/Main.kt [hunk 1]
```diff
@@ -152,9 +152,14 @@ class Main : Runnable {
       inputPublisher = LineInputPublishers(outputHandler)
     }
 
+    if (route != null) {
+      metadataFormat = "composite"
```
Comment (yschimke) [inline src/main/kotlin/io/rsocket/cli/Main.kt]: @linux-china What are firm assumptions we can make when route is selected?

Comment (linux-china) [reply src/main/kotlin/io/rsocket/cli/Main.kt]: it's a nice to make Spring route simple.  A question, I know some developers use cbor for data en/decoding with RSocket.  Could you comment  dataFormat assign(json by default now) when route is not null, and we can use route and cbor both. thanks. 

```
    if (route != null) {
      metadataFormat = "composite"
      // dataFormat = "json
 }
```

Comment (yschimke) [reply src/main/kotlin/io/rsocket/cli/Main.kt]: Should I remove binary as the default?  Or check for it and then override to json?

I'd like to get it down to the minimal set of command line arguments to activate this.

Comment (linux-china) [reply src/main/kotlin/io/rsocket/cli/Main.kt]: for binary, could we consider base64 text for cbor and other binary?

Comment (linux-china) [reply src/main/kotlin/io/rsocket/cli/Main.kt]: binary as base64 text just for reference to implement CLI.  we can consider this feature later. Now json is good for CLI to Spring RSocket, and Spring RSocketStrategies supports to config multi encoders.

**Review by linux-china — APPROVED** [review]

File: src/main/kotlin/io/rsocket/cli/Main.kt (hunk 1 above)
Comment (yschimke) [reply src/main/kotlin/io/rsocket/cli/Main.kt]: OK, I'm only setting when unset.  Take a look.

**Pull request merged**